docs/src/Live.js

```diff
@@ -22,7 +22,7 @@ import { photo } from './constants'
 
 const { colors } = theme
 
-const _scope = Object.assign({}, Rebass, { styled, photo })
+const _scope = { ...Rebass, ...{ styled, photo } }
```

Comment:
The nested object syntax here isn't needed. Mind changing these to only spread objects? e.g. `{ ...Rebass, styled, photo }`

Reply:
Sure, no problem.

Reply:
Thanks! I was hoping that you could make this on all the changes where there's a nested object, but if you don't have time, I can follow up with another commit